get-involved.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Get involved
+permalink: /get-involved/
+---
+
+Please join us in addressing the housing affordability crisis! Our main organizing platform is [our Discourse](https://discourse.somervilleyimby.org/). It is available through the site, and can optionally be delivered to your inbox.
+
+We also have a [Discord server](https://discord.com/invite/UePgDn7kf7) which is more oriented towards chat.
```

Comment:
I think I’ll remove the invite link, b/c we want folks to go _through_ the Discourse forum (main membership space) and join Discord via the invite link in there.

It gets too tricky when folks only join Discord chat, and not the forum, where we do our “official” organizing.

Reply:
Makes sense! I'll note that I found the Discord somewhat hard to find in Discourse, it was buried in the 10th message of the Welcome thread, so if there's a way to make it more prominent that might be nice. But certainly having the Discourse easier to find is far more important 🙂

Reply:
Ah! Maybe a screenshot with the Discord button circled? It’s on every view in Discourse, top-right next to the Search icon/button and your avatar icon/button/menu.